counsel.el

```diff
@@ -5103,29 +5103,29 @@ in the current window."
 (defvar counsel-compile-history nil
   "History for `counsel-compile'.
 
-This is a list of strings with additional properties which allow the
-history to be filtered depending on the context of the call.  The
-properties include:
+This is a list of strings with additional properties which allow
+the history to be filtered depending on the context of the call.
+The properties include:
 
 `srcdir'
-     the root directory of the source code
+    the root directory of the source code
 `blddir'
-     the root directory of the build (in or outside the srcdir)
+    the root directory of the build (in or outside the `srcdir')
 `recursive'
-     the completion should be run again in `blddir' of this result
+    the completion should be run again in `blddir' of this result
 `cmd'
-     if set only the region with this property will be passed to `compile'
+    if set, pass only the substring with this property to `compile'
 
-If you want to persist history between Emacs sessions you can as this
```

Comment:
Wouldn't it be worth the extra verbiage for those not familiar with what savehist-additional-variables does?

Reply:
Users of `savehist` will know or have to learn how to use it anyway. It would absolutely be overkill and even a nuisance for every variable under the sun to mention that it can be persisted via `savehist`. Personally, I wouldn't mention it all, but there's no harm in keeping this one case. Besides, keeping this last sentence lets the docstring be auto-filled nicely.